pkg/artifacts/paths.go

```diff
@@ -18,14 +18,14 @@ package artifacts
 
 import (
 	"fmt"
-	"os"
-	"path/filepath"
-
 	"github.com/spf13/pflag"
 	"k8s.io/klog"
+	"os"
```

Comment:
nit: stdlib imports should be grouped together at the top, these should not have moved down with third party imports.

it's not _super_ important, but it's the pattern in kubernetes projects, it helps make it clear what the external dependencies are and keeping it this way minimizes the diff